doc/ems-application-guide/src/internal-variables/internal-gains.tex

```diff
@@ -8,15 +8,15 @@ \subsection{People Count Design Level}\label{people-count-design-level}
 
-The internal variable ``Lighting Power Design Level'' provides information about the maximum lighting electrical power level associated with each Lights input object. The units are Watts. This is the value entered in, or resulting from intermediate calculations using, the inputs in the lights object. This variable is useful for scaling the values assigned to the ``Electric Power Level control in the ``Lights'' EMS actuator.
+The internal variable ``Lighting Power Design Level'' provides information about the maximum lighting electrical power level associated with each Lights input object. The units are Watts. This is the value entered in, or resulting from intermediate calculations using, the inputs in the lights object. This variable is useful for scaling the values assigned to the ``Electricity Rate'' control in the ``Lights'' EMS actuator.
 
 \subsection{Plug and Process Power Design Level}\label{plug-and-process-power-design-level}
 
-The internal variable ``Plug and Process Power Design Level'' provides information about the maximum electrical equipment power level associated with each ElectricEquipment input object. The units are Watts. This is the value entered in, or resulting from intermediate calculations using, the inputs in the ElectricEquipment object. This variable is useful for scaling the values assigned to the ``Electric Power Level'' control in the ``ElectricEquipment'' EMS actuator.
+The internal variable ``Plug and Process Power Design Level'' provides information about the maximum electrical equipment power level associated with each ElectricEquipment input object. The units are Watts. This is the value entered in, or resulting from intermediate calculations using, the inputs in the ElectricEquipment object. This variable is useful for scaling the values assigned to the ``Electricity Rate'' control in the ``ElectricEquipment'' EMS actuator.
 
-\subsection{Gas Process Power Design Level}\label{gas-process-power-design-level}
+\subsection{NaturalGas Process Power Design Level}\label{gas-process-power-design-level}
 
-The internal variable ``Gas Process Power Design Level'' provides information about the maximum gas equipment power level associated with each GasEquipment input object. The units are Watts. This is the value entered in, or resulting from intermediate calculations using, the inputs in the GasEquipment object. This variable is useful for scaling the values assigned to the ``Gas Power Level'' control in the ``GasEquipment'' EMS actuator.
+The internal variable ``NaturalGas Process Power Design Level'' provides information about the maximum gas equipment power level associated with each GasEquipment input object. The units are Watts. This is the value entered in, or resulting from intermediate calculations using, the inputs in the GasEquipment object. This variable is useful for scaling the values assigned to the ``NaturalGas Rate'' control in the ``GasEquipment'' EMS actuator.
```

Comment:
Not sure we want to change the internal variable name. Unfortunately (maybe, maybe not) we did not consider changing the object name for GasEquipment to NaturalGasEquipment.  If we keep this one, then we need a transition block for EMS:InternalVariable.

Reply:
So the choices are:
 - Keep this change from Gas to NaturalGas, in which case we need to write a transition rule around it, or
 - Revert this change back to Gas.

Today my vote is to revert this back to Gas.  Let me know if I have this incorrect.